src/Symfony/Component/Yaml/Dumper.php

```diff
@@ -105,7 +105,7 @@ public function dump($input, $inline = 0, $indent = 0, $flags = 0)
                     $blockIndentationIndicator = (' ' === substr($value, 0, 1)) ? (string) $this->indentation : '';
                     $output .= sprintf("%s%s%s |%s\n", $prefix, $dumpAsMap ? Inline::dump($key, $flags).':' : '-', '', $blockIndentationIndicator);
 
-                    foreach (preg_split('/\n|\r\n/', $value) as $row) {
+                    foreach (preg_split('/\n/', $value) as $row) {
```

Comment:
Should we now use `explode()` instead?

Reply:
@xabbuh Just ran the tests after replacing `preg_split('/\n/', $value)` by `explode('\n', $value)` and the tests are failing because the dump isn't the same

Reply:
You need to use double-quotes: `explode("\n", $value)`

Reply:
@xabbuh ok my bad I wasn't using double quotes... tests are fine 😃

Reply:
@xabbuh yes cheers for that, I've pushed the change.